src/lib/operators/export_binary.hpp

```diff
@@ -176,9 +178,67 @@ class ExportBinary : public AbstractReadOnlyOperator {
   template <typename T>
   static void _write_segment(const RunLengthSegment<T>& run_length_segment, std::ofstream& ofstream);
 
-  // Chooses the right FixedSizeByteAlignedVector depending on the attribute_vector_width and exports it.
-  static void _export_attribute_vector(std::ofstream& ofstream, const CompressedVectorType type,
-                                       const BaseCompressedVector& attribute_vector);
+  /**
+   * FrameOfReference Segments are dumped with the following layout:
+   *
+   * Description            | Type                                  | Size in bytes
+   * -----------------------------------------------------------------------------------------
+   * Column Type            | ColumnType                            |   1
+   * Number of Blocks       | uint32_t                              |   4
```

Comment:
Next, I see "Write attribute vector width". Could you please double check this table?